src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -1856,25 +1884,34 @@ private Matches() {}
     });
   }
 
-  public static final Match<Territory> territoryIsBlockadeZone = Match.of(t -> {
-    final TerritoryAttachment ta = TerritoryAttachment.get(t);
-    if (ta != null) {
-      return ta.getBlockadeZone();
-    }
-    return false;
-  });
-
-  public static final Match<UnitType> UnitTypeIsConstruction = Match.of(type -> {
-    final UnitAttachment ua = UnitAttachment.get(type);
-    if (ua == null) {
+  static Match<Territory> territoryIsBlockadeZone() {
+    return Match.of(t -> {
+      final TerritoryAttachment ta = TerritoryAttachment.get(t);
+      if (ta != null) {
+        return ta.getBlockadeZone();
+      }
```

Comment:
This if clause can be merged into a single return statement: `return ta != null && ta.getBlockadeZone();`